db/migrate/20220214195216_create_comments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+
+      t.timestamps
+    end
```

Comment:
According to the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) diagram, the `comment` table has a `text` column. Kindly add this.

`
t.text :text
`

Reply:
Thanks for your suggestion